test/test_helper.rb

```diff
@@ -0,0 +1,13 @@
+require 'minitest'
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+require 'awesome_print'
+# Add simplecov
```

Comment:
Do we need to add simplecov here?

Reply:
It looks like you left out `require 'simplecov'`